### PR TITLE
Discard pagination on location change

### PR DESCRIPTION
**Pull request by s4nji**
Fixes pagination on healthcheck if you navigated from api list page.

### CODE REVIEW

**Comment by carlosyslas** [comment]
Why do we need to clear the pagination state?

**Comment by s4nji** [comment]
Currently pagination state seems to be shared between API List page and Healthcheck page.
Navigating to healthcheck page while having page 5 on API List will open page 5 of healthcheck (which will be an empty page if there's no such page).

**Comment by ivanzusko** [comment]
@s4nji , pagination behaves a bit weird. Steps to reproduce:

1. Initial page (pagination: page 1).
2. go OAuth list (pagination: page 1).
3. HealthCheck list (pagination: page 1).
4. HealthCheck list change pagination to page 2.
5. go OAuth page (pagination: page 1).
6. Go Api list (pagination: **page 2**)

**Comment by ivanzusko** [comment]
@s4nji we need to discard pagination everywhere.
Of course, it's better to make it in one separate place, but, as far as we are going life this week and, as far as the whole pagination will be changed soon, we can discard pagination everywhere.

File: src/store/reducers/healthcheck.reducer.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
     SET_SORTING_FILTER,
     SET_ASCEND_FILTER,
 } from '../constants';
+import { LOCATION_CHANGE } from 'react-router-redux';
```
Comment (ivanzusko) [inline src/store/reducers/healthcheck.reducer.js]: `import`s from libraries better to put before our internal `import`s 😉

**Pull request merged**